src/flags.ts

```diff
@@ -33,6 +33,15 @@ const hub = flags.build({
   },
 });
 
+const branch = flags.build({
+  char: 'B',
```

Comment:
I was wondering why `'B'` instead of `'b'`, but I noticed `'b'` was already used to provide `hub`.
I hope it won't be confusing for users 🤔 
Is it possible to favor something like `'br'` for example ? (I really don't know if it's a good practice to have more than 1 letter)

Reply:
CLI parameters are either short (one char, `-c`) or long (multiple chars, `--chars`). As we already have a `-b`, I'm using `-B`. Anyway, long options are always clearer, and it's documented both in README and in the CLI help.